tests/test_overdrive.py

```diff
@@ -215,14 +229,6 @@ def test_book_info_with_metadata(self):
             sorted(ids)
         )
 
-        # Available formats.
-        [kindle, pdf] = sorted(metadata.formats, key=lambda x: x.content_type)
-        eq_(DeliveryMechanism.KINDLE_CONTENT_TYPE, kindle.content_type)
-        eq_(DeliveryMechanism.KINDLE_DRM, kindle.drm_scheme)
-
-        eq_(Representation.PDF_MEDIA_TYPE, pdf.content_type)
-        eq_(DeliveryMechanism.ADOBE_DRM, pdf.drm_scheme)
```

Comment:
couldn't put it there, because overdrive_availability_information.json doesn't contain formats, but returned the code to the metadata test, now that have meta.circ back.